src/main/java/frc/robot/Robot.java

```diff
@@ -193,6 +198,12 @@ public void autonomousPeriodic() {
      */
     @Override
     public void teleopPeriodic() {
+        //If motor current limiting is active, trigger driver feedback (notify driver that drivetrain power is reduced).
+        if (DriveBase.isMotorLimitActive() == true) {
+            //TODO: Put driver feedback code here. Consult with drive team's preferences.
```

Comment:
Can you put something here as a starting point?

Reply:
OK, will commit new code to provide an example.